mitodl/micromasters
# Updated realistic user data to include users with current country != …

#### What are the relevant tickets?

Closes #813 

#### What's this PR do?

Adds 6 users with a current country that is different from their birth country.

Here are the usernames for those 6 users:

['fake.terri.pena',
 'fake.ted.sullivan',
 'fake.camille.jones',
 'fake.ana.cabrera',
 'fake.alvaro.santana',
 'fake.hugo.suarez']

#### How should this be manually tested?

Delete your current set of fake users and re-generate. Check the generated set for the users mentioned above
